### ISSUE TITLE
Add output directory option to choose where to save --report images

### ISSUE THREAD

**Issue by laurencejackson** [GSTT-CSC/hazen#256]
**Is your feature request related to a problem? Please describe.**
By default each task will save report images to the cwd.

**Describe the solution you'd like**
Add an --output flag to the CLI and backend to choose where to save report images.

We should also modify the output of each task to return the paths to each asset

```python
results = {key: {results: results}, {assets: {image paths: paths}}
```

**Describe alternatives you've considered**
Leaving things saving in the cwd, can get messy in dev environment, or if working out of a single folde.r

**Comment by tomaroberts** [comment]
This this is done with latest release @laurencejackson ?

**Comment by laurencejackson** [comment]
yes, resolved by #266
